alpha/engagements/2024/Rust Foundation/update-2024-11.md

```diff
@@ -0,0 +1,53 @@
+This was a prolific engineering month, particularly on the crates.io side, with improvements to the codebase by moving to `async/await`, additional checks for crate spam and name-squatting, as well as the approach of the finish-line on crates deletion.
+
+## Engineering
+
+### async/await now in crates.io
+
+Tobias and another member of the crates.io team finished the `async/await` migration of the crates.io codebase this month. All API endpoints and background worker tasks are now using asynchronous database queries, and email sending network calls, among other things. This unlocked future performance improvements by now being able to take advantage of the pipelining feature of Postgres database connections.
+
+### Crate deletions
+
+Tobias worked on implementing the "Crate deletion" [RFC](https://github.com/rust-lang/rfcs/pull/3660). He opened a pull request implementing the required API endpoint, and other pull request that introduced a new database table to track previously deleted crates. The latter also allows the crates.io team to delay making crate names available again for re-registration after they were deleted. Tobias also created a [supplemental RFC](https://github.com/rust-lang/rfcs/pull/3731) to just focus on deleting whole crates instead of versions. 
```

Comment:
```suggestion
Tobias worked on implementing the "Crate deletion" [RFC](https://github.com/rust-lang/rfcs/pull/3660). He opened a pull request implementing the required API endpoint, and other pull requests that introduced a new database table to track previously deleted crates. The latter also allows the crates.io team to delay making crate names available again for re-registration after they were deleted. Tobias also created a [supplemental RFC](https://github.com/rust-lang/rfcs/pull/3731) to just focus on deleting whole crates instead of versions. 
```

Reply:
Or maybe "another pull request"?

Reply:
Or maybe "another pull request"?

Reply:
yeah, I think I meant to write "another"